pkg/storage/segments/store.go

```diff
@@ -259,6 +260,12 @@ func (s *segmentStore) Delete(ctx context.Context, path paths.Path) (err error)
 	return s.pdb.Delete(ctx, path)
 }
 
+// Repair retrieves an at-risk segment and repairs and stores lost pieces on new nodes
+func (s *segmentStore) Repair(ctx context.Context, seg *pb.InjuredSegment) (err error) {
```

Comment:
The segment store shouldn't be responsible for repairing the segment. The repairer should do that an afterwards update the segment information.

Reply:
That contradicts this document https://docs.google.com/document/d/17LXgm8vMI0PyZmZ9PpJSl1qDyDxPosQf38A7qEFoYxw/edit#